examples/specs/area_cumulative_freq.vl.json

```diff
@@ -14,7 +14,8 @@
     },
     "y": {
       "field": "Cumulative Count",
-      "type": "quantitative"
+      "type": "quantitative",
+      "aggregate": "max"
```

Comment:
Note that this PR fixes this example which uses `window` transform and plot the chart but without aggregation. The backed data is not aggregated so internally it draws many more unnecessary data points.

Those unnecessary data points break the area chart with the new default stack in area.

Reply:
(nit) A more efficient fix is probably aggregate the count before calling window, but feel free to file a follow up ticket if u don't wanna fix now.